crates/da-clients/ethereum/src/config.rs

```diff
@@ -12,17 +12,11 @@ use crate::EthereumDaClient;
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct EthereumDaConfig {
     pub rpc_url: String,
-    pub memory_pages_contract: String,
```

Comment:
Context on why these were used / are removed now would be appreciated.

Reply:
it's not needed. i guess it was added in the very beginning when it was copied from Madara but never removed it

Reply:
Alright.